src/molecule/scenario.py

```diff
@@ -128,13 +128,19 @@ def prune(self):
         ] + self.config.driver.safe_files
         files = util.os_walk(self.ephemeral_directory, "*")
         for f in files:
-            if not any(sf for sf in safe_files if fnmatch.fnmatch(f, sf)):
-                os.remove(f)
+            try:
+                if not any(sf for sf in safe_files if fnmatch.fnmatch(f, sf)):
```

Comment:
Nit: looks like this condition is actually
```suggestion
                if not any(fnmatch.fnmatch(f, sf) for sf in safe_files):
```